### PR TITLE
Implement tests for particle init and position integrators

### PR DESCRIPTION
**Pull request by mskrip**

### CODE REVIEW

File: integrators_test.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+		lifespan      = 10.0
+		ePosition     = pixel.V(0, 0)
+		eNextPosition = pixel.V(0, 0)
+		eSpeed        = pixel.V(0, 10)
+		eLifespan     = 10.0
+	)
+
+	p := createParticle(pos, nextPos, speed, prevDt, lifespan)
+
+	if p.position != pixel.V(0, 0) {
```
Comment (mitas1) [inline integrators_test.go]: Isn't it be `if p.position != ePosition` ?

File: integrators_test.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+		t.Errorf("Particle initialization: Expected position of %f got %f", ePosition, p.position)
+	}
+
+	if p.nextPosition != pixel.V(0, 0) {
```
Comment (mitas1) [inline integrators_test.go]: The same `p.nextPosition != eNextPosition ` ?

File: integrators_test.go [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+		t.Errorf(
+			"Particle initialization: Expected next position of %f got %f",
+			eNextPosition, p.nextPosition)
+	}
+
+	if p.speed != pixel.V(0, 10) {
```
Comment (mitas1) [inline integrators_test.go]: probably `p.speed != eSpeed`

File: integrators_test.go [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+		t.Errorf("Particle initialization: Expected speed of %f got %f", eSpeed, p.speed)
+	}
+
+	if p.lifespan != 10 {
```
Comment (mitas1) [inline integrators_test.go]: `p.lifespan != eLifespan`

File: integrators_test.go [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+	p := createParticle(pos, nextPos, speed, prevDt, lifespan)
+
+	if p.position != pixel.V(0, 0) {
+		t.Errorf("Particle initialization: Expected position of %f got %f", ePosition, p.position)
+	}
+
+	if p.nextPosition != pixel.V(0, 0) {
+		t.Errorf(
+			"Particle initialization: Expected next position of %f got %f",
+			eNextPosition, p.nextPosition)
```
Comment (mitas1) [inline integrators_test.go]: Is there a better way in go to format this lines? sth like:
```go
t.Errorf(
    "Particle initialization: Expected next position of %f got %f",
    eNextPosition, p.nextPosition
)
```
If yes, than format all occurrence of this.

**Comment by mskrip** [comment]
Fixed

**Comment by mitas1** [comment]
LGTM

**Pull request merged**